client/src/components/ratingsReviews/list/ReviewsTile.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+//maps through all the reviews and formats them into 'tiles'
+export default function ReviewsTile (props) {
+
+  let reviewList = props.review.map( ({
+    review_id,
+    rating,
+    summary,
+    recommend,
+    response,
+    body,
+    date,
+    reviewer_name,
+    helpfulness
+  }) => {
+
```

Comment:
It feels like this line would be more readable if you passed in the review object as the argument to the map() callback (instead of an object literal)...
`props.review.map((review) => {
  ...
  <h3>{review.summary)</h3>
})`
...etc